lifecycle/bin/deploy

```diff
@@ -11,26 +54,26 @@ fi
 
 echo "Deploying $NAMESPACES injected, TLS, and baseline namespaces..."
 
-for i in `seq 1 $NAMESPACES`;
+for i in $(seq 1 "$NAMESPACES");
 do
-  NS=lifecycle$i
-  NS_TLS=$NS-tls
-  NS_BASELINE=$NS-baseline
+  NS="$NS_PREFIX""$i"
+  NS_TLS="$NS"-tls
+  NS_BASELINE="$NS"-baseline
 
   echo "\nDeploying $NS..."
-  kubectl create ns $NS
+  kubectl create ns "$NS"
   cat lifecycle.yml |
-    linkerd inject --linkerd-namespace linkerd-lifecycle - |
-    kubectl -n $NS apply -f -
+    linkerd inject --linkerd-namespace "$CTRL_NS_PREFIX" $INJECT_ARGS - |
+    kubectl -n "$NS" apply -f -
 
   echo "\nDeploying $NS_TLS..."
-  kubectl create ns $NS_TLS
+  kubectl create ns "$NS_TLS"
   cat lifecycle.yml |
-    linkerd inject --linkerd-namespace linkerd-lifecycle-tls --tls optional - |
-    kubectl -n $NS_TLS apply -f -
+    linkerd inject --linkerd-namespace "$CTRL_NS_PREFIX"-tls $INJECT_ARGS - |
```

Comment:
`--tls optional` missing

Reply:
Thanks @siggy, fixed it in 2bf1573ab4f9265d058b2daa6f274f7efa5741c7.